includes/utilities.php

```diff
@@ -85,3 +85,57 @@ function prefix_path( $url ) {
 
 	return $url;
 }
+
+/**
+ * Add a redirect.
+ *
+ * @param array $redirect Arguments for the redirect.
+ *
+ * @return int The ID of the Post if redirect successfully added. Otherwise returns 0.
+ */
+function insert_redirect( $redirect ) {
```

Comment:
I'm not a fan of catch all array arguments in function signatures. I know WordPress uses this extensively, and that's part of why it is, in my humble opinion, such a dumpster fire. I'd rather have separate arguments, which we can then type hint later (i.e. soon), when we make the jump to requiring PHP7.

Reply:
Sure.